apps/web/src/components/virtualized-tree/index.tsx

```diff
@@ -85,8 +85,15 @@ export function VirtualizedTree<T>(props: TreeViewProps<T>) {
     treeRef,
     () => ({
       async refresh() {
-        const children = await getChildNodes(rootId, -1);
-        setNodes(children);
+        // const children = await getChildNodes(rootId, -1);
+        // setNodes(children);
+        const nodes = await fetchChildren(
+          rootId,
+          -1,
+          expandedIds,
+          getChildNodes
+        );
+        setNodes(nodes);
```

Comment:
Why this change?

Reply:
Whenever the tree was `refresh`ed, the children of expanded notebooks weren't loaded. This change fixed that